### PR TITLE
update simplifier distance to reflect zoom

### PR DESCRIPTION
**Pull request by wasp77**
Implements #1981

The distance used in the iterative end-point fit algorithm to
determine if points can be removed no longer ignores the
zoom. As the zoom gets larger this distance will get smaller
and fewer points will be removed, thus making for finer grain
control over the drawing. As the zoom gets smaller the drawing
will get more coarse as more points are removed.

### CODE REVIEW

File: src/components/App.tsx [hunk 1]
```diff
@@ -2885,8 +2885,15 @@ class App extends React.Component<ExcalidrawProps, AppState> {
...
           if (draggingElement.type === "draw") {
+            /**
+             *  the constant .1 is fairly arbitrary but was chosen to keep
+             *  a max distance of 1 (.1 is the farthest zoom out).
+             **/
             mutateElement(draggingElement, {
-              points: simplify([...(points as Point[]), [dx, dy]], 0.7),
+              points: simplify(
+                [...(points as Point[]), [dx, dy]],
+                0.1 / this.state.zoom,
```
Comment (xixixao) [inline src/components/App.tsx:2895]: Can we change this value so that at zoom 1 the granularity stays the same as before?

**Comment by xixixao** [comment]
Awesome @wasp77 ! Let's preserve the current granularity and this is good to go.

**Comment by dwelle** [comment]
And maybe also tweak it for zoom < 1 where it feels the simplification is too lax (esp at max zoom out where it's set to `1`, which AFAIK means it doesn't simplify at all).

**Comment by wasp77** [comment]
> And maybe also tweak it for zoom < 1 where it feels the simplification is too lax (esp at max zoom out where it's set to `1`, which AFAIK means it doesn't simplify at all).

@dwelle I believe it would have simplified still depending on the points selected (so the drawing). Does the change to `.7` address your concerns?

**Comment by xixixao** [comment]
Thanks for the fix!

**Pull request merged**